packages/indexer-integration-tests/tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "rootDir": "tests",
+    "outDir": "dist",
+    "target": "es2020",
+    "module": "commonjs",
+    "declaration": true,
+    "sourceMap": true,
+    "esModuleInterop": true,
+    "strict": true,
+    "composite": true,
+    "lib": ["es2015", "es6", "esnext.asynciterable", "dom", "ES2020.Promise"]
+  },
+  "include": ["tests/**/*.ts", "tests/**/*.d.ts"],
+  "exclude": ["tests/**/__tests__/*.ts"],
+  "references": [{ "path": "../indexer-common" }]
```

Comment:
May want to include references to other indexer packages too.

Reply:
This is copy-paste from another package, but yes definitely. I'll leave this as is to remind about that potential.